Scripts/modelsystem.py

```diff
@@ -97,9 +97,10 @@ def assign_base_demand(self, use_fixed_transit_cost=False):
                 with self.basematrices.open("cost", "aht") as aht_mtx:
                     fixed_cost = aht_mtx["transit"]
             else:
-                self.logger.info("Calculating transit cost")
                 fixed_cost = None
-            self.ass_model.calc_transit_cost(self.zdata_forecast.transit_zone, peripheral_cost, fixed_cost)
+            for transit_class in ["transit_work", "transit_leisure"]:
+                self.logger.info("Calculating transit cost for class {}".format(transit_class))
+                self.ass_model.calc_transit_cost(transit_class, self.zdata_forecast.transit_zone, peripheral_cost, fixed_cost)
```

Comment:
I think this is overkill. Path choices probably do not vary that much over assignment classes that it affect transit fares in any substantial way. Takes long time to run. We could use the same matrix in both classes.

Reply:
https://github.com/HSLdevcom/helmet-model-system/pull/183/commits/4b645a943f21dc623b5eb6f558222759a203acd2